dev/com.ibm.websphere.appserver.features/visibility/protected/io.openliberty.auditCollector.1.0.feature

```diff
@@ -0,0 +1,15 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
```

Comment:
We don't need this new feature. The reason is, you can toleration servlet-5.0 in the existing com.ibm.websphere.appserver.auditCollector-1.0 feature.